coderd/database/queries/templates.sql

```diff
@@ -49,7 +50,9 @@ LIMIT
 	1;
 
 -- name: GetTemplates :many
-SELECT * FROM templates;
+SELECT * FROM templates
+ORDER BY (created_at, id) ASC
```

Comment:
This wasn't relevant for the test failure, but a bonus fix.

Reply:
Does the fake db need this too?

Reply:
I'm guessing not, since they'll be appended to a slice. But I can add it there as well.

Reply:
Yea, we should explicitly sort to match the sql imo. I do this with a copied slice somewhere else in the fake db where ordering matters.

Reply:
c0498a9